templates/ai/what-is-kubeflow.html

```diff
@@ -4,613 +4,321 @@
+          alt="",
+          width="1200",
+          hi_def=True,
+          loading="lazy",
+        ) | safe
+      }}
+    </div>
   </div>
-  <div class="col-8">
-    <h2 class="u-fixed-width u-sv1">Get started today</h2>
-    <p class="u-sv2 p-heading--4">Try-out Kubeflow on your Kubernetes environment. Or on MicroK8s - Zero-ops Kubernetes with high availability.</p>
-    <p>Deploy locally on your workstation, cloud VM or on-prem server.</p>
-    <p>
-      <a class="p-button--positive" href="https://charmed-kubeflow.io/docs/quickstart">
-      Try Kubeflow on MicroK8s
-      </a>
-      <a href="https://charmed-kubeflow.io/#get-in-touch">
-        Contact us now&nbsp;&rsaquo;
-      </a>
-    </p>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <h2 class="col">Who uses Kubeflow?</h2>
+    <div class="col p-section">
+      <p>Thousands of companies have chosen Kubeflow for their AI/ML stack.</p>
+      <p>From research institutions like CERN, to transport and logistics companies - Uber, Lyft, GoJek - to financial
+        and media industries with Spotify, Bloomberg, Shopify and PayPal.</p>
+      <p class="p-section--shallow">Forward-looking enterprises are using Kubeflow to empower their data scientists.</p>
+      <hr />
+      <a href="https://ubuntu.com/engage/ai-ml-casestudy-tasmania/">Read our case study with
+        the University of Tasmania</a>
+    </div>
+    <div class="p-logo-section">
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/ea6008a0-spotify-logo.png" alt="Spotify" class="p-logo-section__logo" />
+        </div>
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/b6b0b414-uber-logo.png" alt="Uber" class="p-logo-section__logo" />
+        </div>
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/63870c2f-paypal-logo.png" alt="PayPal" class="p-logo-section__logo" />
+        </div>
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/6117dd11-lyft-logo.png" alt="Lyft" class="p-logo-section__logo" />
+        </div>
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/c358b12a-cern-logo.png" alt="Cern" class="p-logo-section__logo" />
+        </div>
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/b0eca934-Bloomberg-Logo.png" alt="Bloomberg" class="p-logo-section__logo" />
+        </div>
+        <div class="p-logo-section__item">
+          <img src="https://assets.ubuntu.com/v1/a1996ad2-shopify-logo.png" alt="Shopify" class="p-logo-section__logo" />
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <h2 class="col u-no-margin--bottom">Kubeflow resources</h2>
```

Comment:
please don't do it like this, i.e. don't combine functional tags with classes. create a separate div with "col" class and have h2 as a child. Please do the same below too.